kitchen-sink/client/imports/components/ImageDemo.jsx

```diff
@@ -11,8 +11,8 @@ export default class ImageDemo extends React.Component {
 
     const wrapperProps = {
       style: {
-        width: '100%',
```

Comment:
Leave is like this, do not change to fixed with and height. It does not make too much sense to have a fixed with/height as default. Anyway user is going to override it.